app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -103,8 +104,8 @@ class AuthenticationFragment : Fragment(), AuthenticationView {
     }
 
     override fun showError(message: Int) {
-        if (activity?.isFinishing != false) {
```

Comment:
I don't think we want to remove the check for `isFinishing`

Reply:
isFinishing is only useful when, the user hits the back button or Activity#finish() has explicitly been called. There are more ways an activity or fragment can get destroyed. So I think using isFinishing in this situation doesn't suffice. [isStarted](https://developer.android.com/reference/kotlin/androidx/lifecycle/Lifecycle.State#started) will already be false when it reached onPause.

Reply:
Totally missed that it covered that case as well.